e2e/evm/test.go

```diff
@@ -36,49 +36,67 @@ func SetupEVM2EVEMTestSuite(fabric1, fabric2 calls.TxFabric, endpoint1, endpoint
 
 type IntegrationTestSuite struct {
 	suite.Suite
-	client            TestClient
-	client2           TestClient
-	bridgeAddr        common.Address
-	erc20HandlerAddr  common.Address
-	erc20ContractAddr common.Address
-	fabric1           calls.TxFabric
-	fabric2           calls.TxFabric
-	endpoint1         string
-	endpoint2         string
-	adminKey          *secp256k1.Keypair
+	client             TestClient
+	client2            TestClient
+	gasPricer          calls.GasPricer
+	bridgeAddr         common.Address
+	erc20HandlerAddr   common.Address
+	erc20ContractAddr  common.Address
+	genericHandlerAddr common.Address
+	assetStoreAddr     common.Address
+	fabric1            calls.TxFabric
+	fabric2            calls.TxFabric
+	endpoint1          string
+	endpoint2          string
+	adminKey           *secp256k1.Keypair
+	erc20RID           [32]byte
```

Comment:
`erc20RID`
`genericRID`

these resource ID types should probably also reflect changes made from above --> `internalTypes.ResourceID`, if changes are made